src/ansys/platform/instancemanagement/client.py

```diff
@@ -263,3 +265,36 @@ def create_instance(
             )
         definition = definitions[0]
         return definition.create_instance(timeout=requests_timeout)
+
+    def get_instance(self, name: str, timeout: float = None) -> Instance:
+        """Get a remote product instance by name.
+
+        Parameters
+        ----------
+        name: str
+            Name of the instance to get (for example ``instances/mapdl-1212``).
+
+        timeout : float, optional
+            Maximum time in seconds for the request. The default is ``None``.
+
```

Comment:
The default is ``None``, in which case ? where ? is something like "there is no maximum time out". What happens when the default is used?

Reply:
The above comment is true for all defaults where the behavior is not obvious. This is almost always true when the default is ``None``.`

Reply:
Yes, I agree, I've created #50 to track this